First_section.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="First_section.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <header class="toolbar">
+        <h1 id="Logo">My Logo</h1>
+        <img id="menu" src="union.png" />
+      </header>
+
+      <main class="content">
+        <span class="text_header">
+          <p1 id="welcome"> I am Christian Glad to see you! </p1>
```

Comment:
- [ ] No tag in html called p1, the best to be used here is h1

Reply:
changed to <p>